src/addons/attach/attach.js

```diff
@@ -11,12 +11,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = attach(require('../../dist/xterm'));
+    module.exports = attach(require('../../xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../dist/xterm'], attach);
+    define(['../../xterm'], attach);
```

Comment:
@parisk I didn't want to push this into typescript_build_2 until https://github.com/sourcelair/xterm.js/pull/296 is fine to merge as well as I don't want to make the diff hard to read if it still needs reviewing. What were your thoughts on https://github.com/sourcelair/xterm.js/issues/335?